streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -859,7 +859,7 @@ private boolean addClientAssignments(final Set<TaskId> statefulTasks,
             final List<TopicPartition> activePartitionsList = new ArrayList<>();
             final List<TaskId> assignedActiveList = new ArrayList<>();
 
-            final Set<TaskId> activeTasksRemovedPendingRevokation = populateActiveTaskAndPartitionsLists(
```

Comment:
odd git diff?

Reply:
Fixed a minor spelling error. The only changes in this file are this renaming, and removing an instance of unnecessary `toString()` on an argument to `String.format()`